charts/minio/1.3.11/templates/deployment.yaml

```diff
@@ -46,8 +46,8 @@ spec:
             {{ $envList = mustAppend $envList (dict "name" "MINIO_BROWSER_REDIRECT_URL" "value" (printf "%s://%s:%d" (include "minio.scheme" .) .Values.minioDomain (.Values.service.consolePort | int))) }}
             {{ $envList = mustAppend $envList (dict "name" "MINIO_SERVER_URL" "value" (printf "%s://%s:%d" (include "minio.scheme" .) .Values.minioDomain (.Values.service.nodePort | int))) }}
             {{ end }}
-            {{ $envList = mustAppend $envList (dict "name" "MINIO_ACCESS_KEY" "valueFromSecret" true "secretName" $secretName "secretKey" "accesskey") }}
-            {{ $envList = mustAppend $envList (dict "name" "MINIO_SECRET_KEY" "valueFromSecret" true "secretName" $secretName "secretKey" "secretkey") }}
+            {{ $envList = mustAppend $envList (dict "name" "MINIO_ROOT_USER" "valueFromSecret" true "secretName" $secretName "secretKey" "rootUser") }}
```

Comment:
Have you updated the secret where the actual access codes are stored ? As this key won't exist in the secret at all

Reply:
I changed this back to the way it was and I am reducing to merely changing out the environment variables which will silence the error message. And I also changed out the wording of the questions that the users sees, to match what they will see on the login page to the minio ui (username and password not access and secret key), as that might lead to confusion.